### ISSUE TITLE
Trash Cleanup segments not deleting

### ISSUE THREAD

**Issue by ZoombieDyl** [Tercioo/Details-Damage-Meter#91]
When "Auto Erase Trash Segments" under Performace Tweaks is toggled, the trash segments between bosses in mythic pluses do not disappear, I have checked with others and there's do disappear properly but importing their profiles doesn't make the issue fix.

Not sure what I can provide as changing the profile does not seem to remedy it.

Attached is my segment log, the numerous trash cleanups should not be there
https://imgur.com/UH3V2WR

**Comment by Tercioo** [comment]
guild friends have their trash  merged?

------ Original Message ------

**Comment by ZoombieDyl** [comment]
yeah it's just my details that's having this issue, others are having trash merged

**Comment by ZoombieDyl** [comment]
Hi, wondering if there's anything I can provide to help diagnose this issue

**Comment by Tercioo** [comment]
Too much stuff with Beta, didn't have the time to see this yet, sorry.

------ Original Message ------
